README.md

```diff
@@ -51,31 +51,22 @@ If you're using [Maven](https://maven.apache.org/download.html) to manage projec
     <dependency>
         <groupId>com.flowpowered</groupId>
         <artifactId>flow-network</artifactId>
-        <version>1.0.0</version>
+        <version>1.2.4</version>
     </dependency>
 
-If you're using [Gradle](https://www.gradle.org/) to manage project dependencies, simply include the following in your `build.gradle` file:
+If you're using Maven Project to manage project dependencies, simply include the following in your `pom.xml` file:
 
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        compile 'com.flowpowered:flow-network:1.0.0'
-    }
+    <repository>
+        <id>glowstone-repo</id>
+        <url>https://repo.glowstone.net/content/repositories/snapshots/</url>
```

Comment:
I made a mistake here, the URL should be `https://repo.glowstone.net/content/repositories/releases/`